x/evm/statedb/statedb.go

```diff
@@ -396,6 +396,16 @@ func (s *StateDB) AddAddressToAccessList(addr common.Address) {
 	}
 }
 
+// GetAddressToAccessList return full access list
+func (s *StateDB) GetAddressToAccessList() *accessList {
+	return s.accessList
+}
+
+// SetAddressToAccessList overwrite with new access list
+func (s *StateDB) SetAddressToAccessList(accessList *accessList) {
```

Comment:
IMO including "address" here can be confusing
```suggestion
func (s *StateDB) SetAccessList(accessList *accessList) {
```